ocp_resources/network_attachment_definition.py

```diff
@@ -218,14 +219,17 @@ def __init__(
             **kwargs,
         )
         self.network_name = network_name
+        self.overlay_topology = overlay_topology
 
     def to_dict(self):
         super().to_dict()
         if not self.yaml_file:
             if not self.network_name:
                 raise ValueError("network_name is required")
+            if not self.overlay_topology:
+                raise ValueError("overlay_topology is required")
```

Comment:
Is topology required? I didn't find anything about it in the docs.
Also it wasn't required till now, what changed?

Reply:
Before, we had only one type of topology - "layer2". Now we also have a "localnet" topology.
https://docs.openshift.com/container-platform/4.14/networking/multiple_networks/configuring-additional-network.html#configuration-ovnk-additional-networks_configuring-additional-network